packages/aws-cdk/test/api/test.stacks.ts

```diff
@@ -92,9 +92,12 @@ export = testCase({
         Type: 'AWS::CDK::Metadata',
         Properties: {
           Modules: `${require('../../package.json').name}=${require('../../package.json').version}`
-        }
+        },
```

Comment:
Don't modify an existing test case, add a new test case for the new behavior.

Reply:
This test case is `async 'is generated for relocatable stacks'(test: Test) {`... I'm changing the behavior around this particular use-case, effectively breaking this test. It feels modifying the failing test so it matches the new state of the world is... the right thing to do?

Reply:
Cool.